src/features/academic/components/detail/GradeCard.module.scss

```
@@ -29,26 +29,37 @@
 .cardBody {
   display: flex;
   justify-content: space-between;
-  align-items: center;
+  // 과목명이 여러 줄로 길어져도 성적(등급/점수)은 첫 줄에 top 정렬.
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)